### ISSUE TITLE
./waf configure --includes does not add more than one path

### ISSUE THREAD

**Issue by JoostHuizinga** [sferes2/sferes2#46]
When configuring a project, the `--includes` option has some unexpected behavior (at least for me). When adding two paths, only the last path will be included. For example, when configuring with the following command:

`./waf configure --includes /path/to/a --includes /path/to/b `

Sferes will add the following flags:

`CXXFLAGS : -D_REENTRANT -Wall -fPIC -ftemplate-depth-1024 -Wno-sign-compare -Wno-deprecated  -Wno-unused -DSFERES_ROOT="/Users/joost/Documents/sferes/sferes2"  -std=c++11 -I/path/to/b -DEIGEN3_ENABLED `

Notice that `-I/path/to/a` is missing. 

To fix the issue, I would suggest the following modifications to the wscript file in sferes root:

From:
`opt.add_option('--includes', type='string', help='add an include path, e.g. /home/mandor/include', dest='includes')`

To:
`opt.add_option('--includes', action='append', type='string', help='add an include path, e.g. /home/mandor/include', dest='includes')`

And from:
```
    if conf.options.includes :
        conf.env['CXXFLAGS'] += ["-I" + conf.options.includes]
```

To:
```
    if conf.options.includes :
        for inlcude_dir in conf.options.includes:
            conf.env['CXXFLAGS'] += [" -I" + inlcude_dir]
```

**Comment by costashatz** [comment]
I think this is not a bad solution.. If @jbmouret has no objection, create a PR for this... Thanks!

**Comment by costashatz** [comment]
Fixed in #56 ...

**Comment by jbmouret** [comment]
Ok for me.
